contracts/mainnet/connectors/morpho-blue/events.sol

```diff
@@ -0,0 +1,106 @@
+//SPDX-License-Identifier: MIT
+pragma solidity ^0.7.0;
+pragma experimental ABIEncoderV2;
+import "./interface.sol";
+
+contract Events {
+	event LogSupplyAssets(
+		MarketParams marketParams,
```

Comment:
Yes! I was calculating `id()` in the functions only when needed so that's why I didn't add by default. But yes indexing it can be a good option. Updated it [here](https://github.com/Instadapp/dsa-connectors-2.0/pull/1/files#diff-aede42a03ff380c808f4cd96825617617b1a89d36ca95f936fcfe5af64e85f00R7)

Reply:
Resolved! You could also index `onBehalf` and `receiver` whenever it makes sense to you
I'm also pointing out that you do not log `caller = msg.sender` but it may be a design choice from you!